.ci/wercker.sh

```diff
@@ -447,6 +447,16 @@ no-warning-imports-java-design-patterns)
   fi
   ;;
 
+validate-ci-temp-empty)
+  fail=0
+  if [ "$(ls -A .ci-temp)" ]; then
+    ls -A .ci-temp
+    echo ".ci-temp/ is not empty. Verification failed."
```

Comment:
Please add some display for pass as well as failure. CI doesn't say anything if it is correctly empty. It is better to display more than less as when logic starts breaking we will need the displays to help us debug what went wrong.

Is it possible to display the `ls` regardless if it is empty or not?

Reply:
No need to apologize, I've been super busy with school, anyway. That's a good point, CI should acknowledge the successful validation.  Thank you. Instead of `ls` 'ing an empty directory, though, I've emulated the tests above this one in the bash script, so that bash will echo `Empty .ci-temp/ validation did not find any warnings.`  if validation is successful.  If you would rather have just ls, let me know and I'll make the changes.

Reply:
Like I said more is better. I would display both the ls and the message.

Reply:
Done.